src/coreclr/vm/wasm/cgencpu.h

```diff
@@ -22,8 +23,8 @@
 
 inline unsigned StackElemSize(unsigned parmSize, bool isValueType = false /* unused */, bool isFloatHfa = false /* unused */)
 {
-    _ASSERTE("The function is not implemented on wasm");
-    return 0;
+    const unsigned stackSlotSize = INTERP_STACK_SLOT_SIZE;
+    return ALIGN_UP(parmSize, stackSlotSize);
```

Comment:
StackElemSize is used to compute the amount of memory used to pass argument in native ABI. It does not make sense to me to tie it together with interpreter. I think it would be better to keep it as is with assert and add just do a local fix in callingconvention.h that's with the interpreter calling convention currently.